exercises/queen-attack/src/example/java/QueenAttackCalculator.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Arrays;
+
+public final class QueenAttackCalculator {
+
+    private final int[] queen1Coordinates;
+
+    private final int[] queen2Coordinates;
+
+    public QueenAttackCalculator(final int[] queen1Coordinates, final int[] queen2Coordinates)
```

Comment:
I know that the README doesn't call it out, but what do you think of naming the parameters `blackQueenCoordinates` and `whiteQueenCoordinates`?

Personally, I find exercises that immerse me in the domain a bit more engaging... as if I'm solving a trimmed down version of "the real thing."